src/core/display/Container.js

```diff
@@ -64,18 +74,14 @@ export default class Container extends DisplayObject
 
             child.parent = this;
 
-            // ensure a transform will be recalculated..
-            this.transform._parentID = -1;
```

Comment:
Why was this removed? Pretty sure it is necessary to reset the transform/bounds Ids when adding the child to a new parent.